roles/kubernetes/master/tasks/encrypt-at-rest.yml

```diff
@@ -1,4 +1,25 @@
 ---
+- name: Check if secret for encrypting data at rest already exist
+  stat:
+    path: "{{ kube_cert_dir }}/secrets_encryption.yaml"
+  register: secrets_encryption_file
+
+- name: Slurp secrets_encryption file if it exists
+  slurp:
+    src: "{{ kube_cert_dir }}/secrets_encryption.yaml"
+  register: secret_file_encoded
+  when: secrets_encryption_file.stat.exists
+
+- name: Base 64 Decode slurped secrets_encryption file
+  set_fact:
+    secret_file_decoded: "{{secret_file_encoded['content'] | b64decode | from_yaml}}"
```

Comment:
Slurp fetches the file from the master node and returns it's content encoded in base64. The main reason I did it this way was to avoid having to store the file on the Ansible host node.